### ISSUE TITLE
requestIdleCallback scheduler

### ISSUE THREAD

**Issue by ronag** [Reactive-Extensions/RxJS#1059]
A scheduler based on `requestidlecallback` would be very useful...

**Comment by erykpiast** [comment]
:+1:  (I didn't even know something like this exist, until now)

**Comment by mattpodwysocki** [comment]
@ronag would it make more sense to just put it into the main `Scheduler.default` for Chrome since they are the only ones who implement this?

**Comment by ronag** [comment]
@mattpodwysocki: Yes, actually I think it would make a lot of sense to use it by default.
